packages/appChrome/stories/AppChrome.stories.tsx

```diff
@@ -30,7 +30,7 @@ import {
   blue,
   pink,
   purple
-} from "../../design-tokens/build/js/designTokens";
+} from "@dcos/ui-kit-shared/dist/build/js/designTokens";
```

Comment:
💥 Same as for the styleUtils, but didn't we want to get rid of the sublevel imports?